stages/test/test_erofs.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python3
+
+import os.path
+import subprocess
+from unittest import mock
+
+import pytest
+
+import osbuild.meta
+from osbuild.testutil.imports import import_module_from_path
+
+
+def make_fake_input_tree(tmpdir, fake_content: dict) -> str:
+    basedir = os.path.join(tmpdir, "tree")
+    for path, content in fake_content.items():
+        dirp, name = os.path.split(os.path.join(basedir, path.lstrip("/")))
+        os.makedirs(dirp, exist_ok=True)
+        with open(os.path.join(dirp, name), "w", encoding="utf-8") as fp:
+            fp.write(content)
+    return basedir
+
+
+def have_erofs() -> bool:
```

Comment:
This should just use `testutil.has_executable` from https://github.com/osbuild/osbuild/pull/1438/files#diff-dc512b1b5b8a055b0a21647198912e4ce1df02cacd7343d2b4fb35fba1ab05aeR7 once merged